devtools-panels/devtools/panel/devtools-panel.js

```diff
@@ -0,0 +1,65 @@
+/**
+Handle errors from the injected script.
+Errors may come from evaluating the JavaScript itself
+or from the devtools framework.
+See https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/devtools.inspectedWindow/eval#Return_value
+*/
+function handleError(error) {
+  if (error.isError) {
+    console.log('Devtools error: ${error.code}');
+  } else {
+    console.log('JavaScript error: ${error.value}');
```

Comment:
Changing from `` to '' here and at line 9 introduces an error, please see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals